monkestation/code/modules/research/nanites/nanite_remote.dm

```diff
@@ -19,6 +19,7 @@
 	var/code = 0
 	var/relay_code = 0
 	var/current_program_name = "Program"
+	var/silicon = FALSE		//Affects whether you can lock it, and use local mode.
```

Comment:
Silicons already have AA don't they?

Reply:
They have AA, but they also can't physically have their remote taken from them, so the lock button does nothing of use. I use the silicon variable to disable buttons that would never do anything for them.